cmake/helpers.cmake

```diff
@@ -136,11 +136,17 @@ function(add_umf_target_link_options name)
             endif()
         endif()
     elseif(MSVC)
+        message(STATUS "C_COMPILER_ID: ${C_COMPILER_ID}")
+        message(STATUS "CXX_COMPILER_ID: ${CXX_COMPILER_ID}")
+        message(STATUS "CMAKE_C_COMPILER_ID: ${CMAKE_C_COMPILER_ID}")
+        message(STATUS "CMAKE_CXX_COMPILER_ID: ${CMAKE_CXX_COMPILER_ID}")
```

Comment:
nit: These messages potentially can be useful for other CMake generators. Could you move them to some common space, maybe in the main CMakeLists.txt?

Reply:
these were only debug prints - removed